source/ucumLhcUtils.js

```diff
@@ -145,71 +145,157 @@ export class UcumLhcUtils {
   } // end validateUnitString
 
 
+
+/**
+ * @typedef {
+ *   'normal', 
+ *   'mol->mass', 
+ *   'mass->mol', 
+ *   'eq->mass', 
+ *   'mass->eq', 
+ *   'eq->mol', 
+ *   'mol->eq', 
+ * } ConversionType
+ */
+
+  /**
+  * Detects the type of conversion between two units.
+  *
+  * @param {Object} fromUnit - The unit to convert from.
+  * @param {Object} toUnit - The unit to convert to.
+  * @returns {ConversionType} conversionType - The type of conversion as a string.
+  */
+  detectConversionType(fromUnit, toUnit) {
+    /** @type {ConversionType} */
+    let conversionType = 'normal';
+
+    /** Note: 
+     * isMolarUnit() will return true for both molar and equivalent units 
```

Comment:
It doesn't look like that is true.  fromUnit.isMolarUnit only check moleExp_, not equivalentExp_ (which I think is what it should do).

Reply:
AFAIK, equivalents _are_ molar units. Any eq unit should (and currently does) have both moleExp_ !== 0 and equivalentExp_ !== 0.

The definition of an equivalent is relative to moles - this can be seen reflected in the definition files:

```xml
   <unit Code="eq" CODE="EQ" isMetric="yes" class="chemical">
      <name>equivalents</name>
      <printSymbol>eq</printSymbol>
      <property>amount of substance</property>
      <value Unit="mol" UNIT="MOL" value="1">1</value>
   </unit>
```

For example, this test currently passes:
```js
    it("equivilant should have both molExp_ and equivalentExp_ set", function(){
      var eqUnit = uTabs.getUnitByCode('eq');
      assert.equal(eqUnit.moleExp_, 1);
      assert.equal(eqUnit.equivalentExp_, 1);
    });
```

I do believe this should be the correct behavior.